graph/src/data/subgraph/schema.rs

```diff
@@ -0,0 +1,343 @@
+//! See `core/src/subgraph/subgraph.graphql` for corresponding graphql schema.
+
+use super::SubgraphId;
+use components::store::{EntityKey, EntityOperation, Store};
+use data::store::Value;
+use failure::Error;
+use std::collections::HashMap;
+use uuid::Uuid;
+
+/// ID of the subgraph of subgraphs.
+pub const SUBGRAPHS_ID: &str = "__subgraphs";
+const EVENT_SOURCE: &str = "SubgraphAdded";
```

Comment:
And I think this one we should just leave empty for now.

Reply:
Ok, I'm coming around to that (it does make sense 😉). `subgraph-added` perhaps? (I'm a fan of separating things with dashes, blame Clojure.)

Reply:
Clojure style it is.